projects/plugins/jetpack/changelog/add-calendly-tests-and-fixtures

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
Tiny nit: should this be `patch` instead of minor since it isn't adding additional functionality?

Reply:
It could be. Maybe I was taking the prompts in changelogger too literally. `patch` was for backwards compatible bug fixes that this isn't really. I chose minor as it's closer to new functionality than a bug fix?

<img width="755" alt="Screen Shot 2021-03-18 at 1 15 54 pm" src="https://user-images.githubusercontent.com/60436221/111568336-4d122b00-87ec-11eb-9672-cc5c0ba7c3f3.png">

Reply:
Good point! I'd be happy to leave it :)

Reply:
You'll be happy to hear that `patch`, `minor`, and `major` are not that important for the Jetpack plugin, since we don't follow semver for the plugin. :) We follow WP Core's example there (going from 9.9 to 10.0 when releasing a new version, regardless of the type of change).

The significance of the changes is more important for the other projects in the Jetpack monorepo; our Composer packages all use semver and we use the info provided in each PR's changelog entry to automatically decide how to increment the version number in the next version of the package.